### PR TITLE
Update all packages for isolated development

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

**What was changed:**
This PR updates the package setup for the individual packages so that they can be independently worked on. This will allow engineers to go into a package and install all the needed devDependencies and run scripts for development as an alternative to doing them at the root level.

**Why it was changed:**
This change is so that developers can work on packages in isolation if they may choose so. Working on a package in isolation is useful as that developers can ignore the other packages that are not part of their current work. And it results in faster installation and test execution which may help speed up development. For example, running `npm install` may take a few minutes at the repo level but <30seconds at the package level. 

This will also help maintenance in the long run by making it clear what dev tools are needed for each individual package. Relying solely on the root level obfuscates which devDependencies are needed for a particular package and can make it difficult to relocate or move that package to another repo.

**Note**
There will be additional PRs to help refine this setup as the work effort to properly isolate packages is considerable. The current status of the packages are:
- everything works as intended at the repo root level
- linter runs for every package
- jest tests work for all packages except `terra-functional-testing`
 
**Future Work**
- update `terra-functional-testing` so that it can be worked properly in isolation. This may require source code updates as some of the features rely on hard-coded paths.
- update `terra-toolkit-docs` so that the dev site can be run from both, the root level and the package.
- clean up dependencies at the root level.

### Testing

This change was tested using:

- [ ] WDIO
- [x] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [x] Other (please describe below)
- [ ] No tests are needed

This was tested by running the linter and jest tests for each individual package and at the root level to ensure the existing setup is not broken.

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-functional-testing/tests/jest/commands/utils/eventEmitter.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import eventEmitter from '../../../../src/commands/utils/eventEmitter'; // eslint-disable-line import/no-duplicates
-import eventEmitter2 from '../../../../src/commands/utils/eventEmitter'; // eslint-disable-line import/no-duplicates
+const eventEmitter = require('../../../../src/commands/utils/eventEmitter'); // eslint-disable-line import/no-duplicates
```
Comment (sycombs) [inline packages/terra-functional-testing/tests/jest/commands/utils/eventEmitter.test.js:1]: Just curious, why do we need to change to `require` syntax here?

**Review by sycombs — APPROVED** [review]
Looks good to me.

File: packages/terra-functional-testing/tests/jest/commands/utils/eventEmitter.test.js (hunk 1 above)
Comment (sdadn) [reply packages/terra-functional-testing/tests/jest/commands/utils/eventEmitter.test.js:1]: For some reason, babel was having issues with compiling `import` at the package level while mysteriously working at the root level. I had theoretically the same config on both levels so I couldn't figure out what's going on. However during debugging, I discover that most of the terra-functional-tests tests utilize `require` and only the ones with `import` were causing issues. So I changed these for consistency and it resolved the babel issue.

**Review by artpark — APPROVED** [review]

**Comment by kenk2** [comment]
> Fails
> 🚫	Please include a CHANGELOG entry for each changed package on this PR. Looks like a CHANGELOG entry is missing for:
> * terra-aggregate-translations
> * terra-toolkit-docs
> * browserslist-config-terra
> * duplicate-package-checker-webpack-plugin
> * eslint-config-terra
> * jest-config-terra
> * package-json-lint-config-terra
> * package-json-lint
> * stylelint-config-terra
> * terra-cli
> * terra-enzyme-intl
> * terra-functional-testing
> * terra-open-source-scripts
> * webpack-config-terra
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [544207c](https://github.com/cerner/terra-toolkit/commit/544207c268be60d082bc48da6039cf665c252741)

No consumer-related changes, no changelog is necessary.

**Pull request merged**